src/pages/Addons/components/detail/index.tsx

```diff
@@ -308,6 +368,33 @@ class AddonDetailDialog extends React.Component<Props, State> {
             </Card>
           </Loading>
         </DrawerWithFooter>
+
+        <Dialog
+          locale={locale.Dialog}
+          visible={showStatusVisible}
+          className={'commonDialog'}
+          title={<Translation>Addons Status</Translation>}
+          autoFocus={true}
+          isFullScreen={true}
+          style={{ width: '800px' }}
+          onClose={this.onStatusClose}
+          footer={
+            <div className="next-dialog-footer">
+              <Button onClick={this.onStatusClose}>
+                <Translation>Close</Translation>
+              </Button>
+              <Button type="primary" onClick={this.loadAddonStatus}>
+                <Translation>Refresh</Translation>
+              </Button>
+            </div>
+          }
+          height="auto"
+          footerAlign="center"
+        >
+          <Loading visible={statusLoading} style={{ width: '100%' }}>
+            <Status applicationStatus={addonsStatus} />
```

Comment:
Include Dialog in  Status component?

Reply:
Extract common components to deal with different status details dialog